parlai/mturk/core/socket_manager.py

```diff
@@ -197,7 +199,9 @@ def __init__(self, server_url, port, alive_callback, message_callback,
         self.queues = {}
         self.threads = {}
         self.run = {}
-        self.last_heartbeat = {}
+        self.last_sent_heartbeat = {}  # time of last heartbeat sent
+        self.last_gotten_heartbeat = {}  # actual last received heartbeat
```

Comment:
nit: maybe `last_received_heartbeat`? something about `last_gotten` sounds awk to me (and in the comment you write received)